src/java/org/apache/cassandra/net/RequestCallbacks.java

```diff
@@ -112,6 +119,11 @@ public void addWithExpiration(AbstractWriteResponseHandler<?> cb,
                                   boolean allowHints)
     {
         assert message.verb() == Verb.MUTATION_REQ || message.verb() == Verb.COUNTER_MUTATION_REQ || message.verb() == Verb.PAXOS_COMMIT_REQ;
+        if (shutdown)
+        {
+            logger.debug("Received request after messaging service shutdown so ignoring it");
```

Comment:
Should this be at the `TRACE` level? It's possible we'll swamp the logs if we print at `debug`.

Reply:
I still think we want to avoid hitting this debug message in production so that we don't fill the logs with noise. I can imagine we'd see way more than 400 messages if a pod dies/gets bounced.